src/components/PersonLink.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  return (
+    <>
+      <Link
+        className={classNames({ 'has-text-danger': person.sex === 'f' })}
+        to={`/people/${person.slug}`}
+      >
+        {person.name}
+      </Link>
+    </>
+  );
```

Comment:
Fragment is redundant
```suggestion
  return (
      <Link
        className={classNames({ 'has-text-danger': person.sex === 'f' })}
        to={`/people/${person.slug}`}
      >
        {person.name}
      </Link>
  );
```